index.bs

```diff
@@ -0,0 +1,225 @@
+urlPrefix: https://w3c.github.io/resource-timing/; spec: RESOURCE-TIMING;
+    type: dfn; url: #sec-privacy-security; text: statistical fingerprinting;
+urlPrefix: https://w3c.github.io/hr-time/; spec: HR-TIME-2;
+    type: typedef; url: #idl-def-domhighrestimestamp; text: DOMHighResTimeStamp;
+    type: interface; url: #dfn-performance; text: Performance;
+    type: method; for:Performance;
+        text: now(); url: #dom-performance-now;
+    type: dfn; text: current high resolution time; url: #dfn-current-high-resolution-time;
+    type: attribute; for: WindowOrWorkerGlobalScope;
+        text: performance; url: #dom-windoworworkerglobalscope-performance;
+urlPrefix: https://www.w3.org/TR/CSS21/visuren.html; spec: CSS21;
+    type: dfn; url: #viewport; text: viewport; 
+urlPrefix: https://w3c.github.io/paint-timing/; spec: PAINT-TIMING;
+    type: dfn; url: #mark-paint-timing; text: mark paint timing;
+urlPrefix: https://www.w3.org/TR/css-box-3/; spec: CSS-BOX-3;
+    type: dfn; url: #border-box; text: border box; 
+urlPrefix: https://www.w3.org/TR/css-break-3/; spec: CSS-BREAK-3;
+    type: dfn; url: #box-fragment; text: box fragment;
+urlPrefix: https://www.w3.org/TR/cssom-view-1/; spec: CSSOM-VIEW-1;
+    type: dfn; url: #css-pixels; text: CSS pixels;
+urlPrefix: https://www.w3.org/TR/css-values-4/; spec: CSS-VALUES-4;
+    type: dfn; url: #pixel-unit; text: pixel units;
+urlPrefix: https://wicg.github.io/visual-viewport/index.html; spec: VISUAL-VIEWPORT;
+    type: dfn; url: #dom-visualviewport-width; text: visual viewport width;
+    type: dfn; url: #dom-visualviewport-height; text: visual viewport height;
+urlPrefix: https://www.w3.org/TR/CSS2/visudet.html; spec: CSS2;
+    type: dfn; url: #containing-block-details; text: initial containing block;
+    
+
+
+</pre>
+
+Introduction {#sec-intro}
+=====================
+
+<div class="non-normative">
+
+<em>This section is non-normative.</em>
+
+When the layout of a page is unstable it leads to a very bad user experience.
+This shifting of DOM elements is often due to content loading asynchronously and displacing other elements on the page.
+The layout instability metric identifies these unstable pages by measuring the layout shift fraction of each animation frame on the page, allowing the user to compute an overall instability score for the page.
+
+</div>
+
+Usage example {#example}
+------------------------
+
+<pre class="example highlight">
+    var observer = new PerformanceObserver(function(list) {
+        var perfEntries = list.getEntries();
+        for (let i = 0; i < perfEntries.length; i++) {
+            // Process entries
+            // Report back for analytics and monitoring
+            // ...
+        }
+    });
+
+    // Register observer for layout shift notifications
+    observer.observe({entryTypes: ["layoutShift"]});
+</pre>
+
+<div class="non-normative">
+
+<em>This section is non-normative.</em>
```

Comment:
Which section? Normally this would go at the beginning of a section.

Reply:
The introduction section.